x/wasm/internal/keeper/reflect_test.go

```diff
@@ -451,10 +439,14 @@ type maskCustomMsg struct {
 	Raw   []byte `json:"raw,omitempty"`
 }
 
-// toMaskRawMsg encodes an sdk msg using amino json encoding.
+// toMaskRawMsg encodes an sdk msg using any type with json encoding.
```

Comment:
woah, this is working again.
that is a nice feat. I would have probably removed it, but nice one